modules/mvt/src/mvt-loader.js

```diff
@@ -16,7 +16,8 @@ export const MVTWorkerLoader = {
     mvt: {
       coordinates: 'local',
       tileIndex: {},
-      workerUrl: `https://unpkg.com/@loaders.gl/mvt@${VERSION}/dist/mvt-loader.worker.js`
+      workerUrl: `https://unpkg.com/@loaders.gl/mvt@${VERSION}/dist/mvt-loader.worker.js`,
+      layerProperty: 'layerName'
```

Comment:
Nit; Order of props seems a little random. Maybe alphabetical order (perhaps with the cross-loader `workerUrl`) prop at the top or bottom would look a little cleaner?